dds/src/Conversion.cpp

```diff
@@ -19,6 +19,9 @@
 #include <sstream>
 #include <soss/utilities.hpp>
 #include <fastrtps/types/MemberDescriptor.h>
+#if WIN32
+#include <Windows.h>
```

Comment:
For what is this needed?

Reply:
For definition of u_char. But I think is defined in winshock so I'm going to change that include (Windows includes winsock).

Reply:
Done

Reply:
I want to Jaime see this `u_char` usage. I assign him.

Reply:
I've changed this u_char for an uint8_t